Stack/Opc.Ua.Core/Stack/Server/EndpointBase.cs

```diff
@@ -822,6 +823,96 @@ public static IServiceRequest GetRequest(IAsyncResult ar)
             #endregion
 
             #region Private Members
+
+            /// <summary>
```

Comment:
I twere better to have the tracecontext in a seperate class in this location.

Reply:
Given that the TraceContext class has been removed, and only the Extract method remains, as discussed, it would be better to retain the extraction functionality within the EndpointBase